project/Publish.scala

```diff
@@ -21,12 +21,17 @@ object Publish extends AutoPlugin {
   import bintray.BintrayPlugin
   import bintray.BintrayPlugin.autoImport._
 
+  object CliOptions {
+    val publishRepo = CliOption("akka.publish.repo", "maven")
+  }
+
   override def trigger = allRequirements
   override def requires = BintrayPlugin
 
   override def projectSettings = Seq(
     bintrayOrganization := Some("akka"),
-    bintrayPackage := "com.typesafe.akka:akka-http_2.11"
+    bintrayPackage := "com.typesafe.akka:akka-http_2.11",
```

Comment:
I know it isn't part of this PR, but is `_2.11` correct here?

Reply:
Unfortunately, it is. This is the original name of the bintray package that akka-http is deployed to. We can not rename it as it is connected to the way artifacts sync to jcenter.